src/Components/Footer/index.tsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import {
+  MDBFooter,
+  MDBContainer,
+  MDBRow,
+  MDBCol,
+  MDBIcon,
+} from "mdb-react-ui-kit";
+import { t1, t2, t3, t4, t5, t6, t7, t8, t9, t10, t11, t12 } from "./constant";
+export const Footer = () => {
+  return (
+    <MDBFooter className="text-center text-lg-start text-muted bg-dark ">
+      <section className="d-flex justify-content-center justify-content-lg-between p-4 border-bottom">
+        <div className="me-5 d-none d-lg-block">
+          <span>Get connected with us on social networks:</span>
+        </div>
+
+        <div>
+          <a
+            href="https://www.facebook.com/groups/254840054550446"
+            className="me-4 text-reset"
+          >
+            <MDBIcon color="secondary" fab icon="facebook-f" />
+          </a>
+          <a href="https://twitter.com/iskcon_vvn" className="me-4 text-reset">
+            <MDBIcon color="secondary" fab icon="twitter" />
+          </a>
+          <a href="#" className="me-4 text-reset">
+            <MDBIcon color="secondary" fab icon="google" />
+          </a>
+          <a href="#" className="me-4 text-reset">
+            <MDBIcon color="secondary" fab icon="instagram" />
+          </a>
+          <a href="#" className="me-4 text-reset">
+            <MDBIcon color="secondary" fab icon="linkedin" />
+          </a>
+          <a href="https://t.me/iskconvvn" className="me-4 text-reset">
+            <MDBIcon color="secondary" fab icon="telegram" />
+          </a>
```

Comment:
Can we use the map function here?

Reply:
yes, I have updated it using map function